src/olympia/versions/migrations/0009_auto_20200717_1137.py

```diff
@@ -2,7 +2,6 @@
 
 from django.db import migrations, models
 
-from olympia.constants.promoted import RECOMMENDED
```

Comment:
Huh. Does the linter not check migrations?

Reply:
Correct.. it's terrible. Like probably one of the worst possible things.